scripts/currently_disabled/error_checking_prompt.py

```
@@ -0,0 +1,215 @@
+        
+        for file_pattern, suggestion in deprecated_patterns.items():
+            for py_file in self.root_dir.rglob("*.py"):
+                if file_pattern in str(py_file):
+                    try:
+                        with open(py_file, 'r', encoding='utf-8') as f:
+                            content = f.read()
+                        if file_pattern.replace("/", ".").replace(".py", "") in content:
+                            warning_msg = f"Warning: {py_file} uses deprecated code. Use {suggestion}"
+                            print(warning_msg)
+                            # This is a warning, not an error
+                    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)